step-plans/step-plans-core/src/main/java/step/core/artefacts/reports/resolvedplan/ResolvedPlanBuilder.java

```diff
@@ -36,35 +37,37 @@ public ResolvedPlanBuilder(ExecutionContext executionContext) {
     }
 
     public ResolvedPlanNode buildResolvedPlan(Plan plan) {
-        return buildTreeRecursively(null, plan.getRoot(), null, plan, ParentSource.MAIN);
+        return buildTreeRecursively(null, plan.getRoot(), null, plan, ParentSource.MAIN ,1);
```

Comment:
- We're setting the position "1" for the first item here. Shouldn't it be 0 as for the others below?
- Formatting